scripts/variantstore/wdl/GvsCreateVAT.wdl

```diff
@@ -105,58 +95,41 @@ workflow GvsCreateVAT {
 
 ################################################################################
 
-task MakeSubpopulationFiles {
+task MakeSubpopulationFilesAndReadSchemaFiles {
     input {
         File input_ancestry_file
-        String? service_account_json_path
         File inputFileofFileNames
         File inputFileofIndexFileNames
-    }
-    parameter_meta {
-        input_ancestry_file:
-        {
-            localization_optional: true
-        }
-        inputFileofFileNames:
-        {
-            localization_optional: true
-        }
-        inputFileofIndexFileNames:
-        {
-            localization_optional: true
-        }
+
+        String vat_schema_json_filepath = "/data/variant_annotation_table/schema/vat_schema.json"
+        String variant_transcript_schema_json_filepath = "/data/variant_annotation_table/schema/vt_schema.json"
+        String genes_schema_json_filepath = "/data/variant_annotation_table/schema/genes_schema.json"
     }
     String output_ancestry_filename =  "ancestry_mapping.tsv"
     String custom_annotations_template_filename =  "custom_annotations_template.tsv"
-    String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
-    String updated_input_ancestry_file = basename(input_ancestry_file)
-    String updated_input_vcfs_file = basename(inputFileofFileNames)
-    String updated_input_indices_file = basename(inputFileofIndexFileNames)
+
+    String vat_schema_json_filename = basename(vat_schema_json_filepath)
+    String variant_transcript_schema_json_filename = basename(variant_transcript_schema_json_filepath)
+    String genes_schema_json_filename = basename(genes_schema_json_filepath)
 
     command <<<
         set -e
 
-        if [ ~{has_service_account_file} = 'true' ]; then
-            gsutil cp ~{service_account_json_path} local.service_account.json
-            export GOOGLE_APPLICATION_CREDENTIALS=local.service_account.json
-            gcloud auth activate-service-account --key-file=local.service_account.json
-        fi
-
-        gsutil cp ~{input_ancestry_file} .
-        gsutil cp ~{inputFileofFileNames} .
-        gsutil cp ~{inputFileofIndexFileNames} .
+        cp ~{vat_schema_json_filepath} ~{vat_schema_json_filename}
+        cp ~{variant_transcript_schema_json_filepath} ~{variant_transcript_schema_json_filename}
+        cp ~{genes_schema_json_filepath} ~{genes_schema_json_filename}
```

Comment:
Is this just `cp`ing all these to `.`? Could all of this be simplified to something like

```shell
ln /data/variant_annotation_table/schema/* .
```

Reply:
Had to go back to using a `cp` - even `ln -s` wouldn't work (cromwell couldn't resolve/delocalize the outputs)